docs/configuration.md

```diff
@@ -1275,6 +1279,21 @@ The `debug` section takes a single, required `addr` parameter. This parameter
 specifies the `HOST:PORT` on which the debug server should accept connections.
 
 
+### headers
+
+The `headers` option is **optional** . Use it to specify headers that the HTTP
+server should include in responses. This can be used for security headers such
```

Comment:
To be very pedantic, we don't include it in all responses. For example, if someone hits a path that has no associated handler, the custom headers won't be included.